docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -6,23 +6,25 @@ Delta Lake connector
 
   <img src="../_static/img/delta-lake.png" class="connector-logo">
 
-The Delta Lake connector allows querying data stored in `Delta Lake
+The Delta Lake connector allows querying data stored in the `Delta Lake
 <https://delta.io>`_ format, including `Databricks Delta Lake
-<https://docs.databricks.com/delta/index.html>`_. It can natively read the Delta
-transaction log, and thus detect when external systems change data.
+<https://docs.databricks.com/delta/index.html>`_. The connector can natively
+read the Delta transaction log and thus detect when external systems change
+data.
 
 Requirements
 ------------
 
-To connect to Databricks Delta Lake, you need:
+To connect to Databricks Delta Lake, you need the following:
```

Comment:
Lets not do this in this PR .. this is the same in all other connectors .. so if we want to change it .. do it in a separate PR for all connectors maybe .. but dont worry for now